thirdparty/fetch-thirdparty-deps.sh

```diff
@@ -94,3 +94,10 @@ if [ ! -f rix0rrr.BeaconLib.dll ]; then
 	cp ./rix0rrr.BeaconLib/lib/net40/rix0rrr.BeaconLib.dll .
 	rm -rf rix0rrr.BeaconLib
 fi
+
+if [ ! -f DiscordRPC.dll ]; then
```

Comment:
We also need to fetch/install/package `Newtonsoft.Json`.

Reply:
Nit: Can we please also put this and Newtonsoft.Json above SDL2-CS, so all the nuget fetches are grouped together?